File: Data/Api.ReactionTracker.Com/Api.ReactionTracker.Com/Models/Reaction.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace Api.ReactionTracker.Com.Models
{
    public class Reaction
    {
        public Reaction()
        {
            Answer1Votes = new List<Vote>();
            Answer2Votes = new List<Vote>();
        }

        [Key]
        public int Id { get; set; }
        public string Question { get; set; }
        public string Answer1 { get; set; }
        public string Answer2 { get; set; }
        public string ImageUrl { get; set; }

        public virtual List<Vote> Answer1Votes { get; set; }
        public virtual List<Vote> Answer2Votes { get; set; }
    }
}

File: Exercises-Completed/Sol5.1/src/components/Reaction.js
import React from 'react'

export default class Reaction extends React.Component{
    render(){

        return(
            <div>
                <h3>{this.props.question}</h3>
                <button>{this.props.answer1}</button>
                <button>{this.props.answer2}</button>
            </div>
        );
    }
}